crypto/__main__.py

```diff
@@ -61,6 +66,24 @@ def main():
             else:
                 print("Input {0} was not a valid file.".format(file_name))
 
+    # NEW - File encryption based on its extension
+    elif files:
+        for filename in files:
+            crypter = Crypter(filename, SeanStrategy())
+            file_contents = read_file(filename)
+            if any(file in filename for file in decryption_extension):
+                print("Decrypting: {0}".format(filename))
+                output_text = crypter.decrypt_txt(file_contents)
+            if any(file in filename for file in encryption_extension):
```

Comment:
```suggestion
            elif any(file in filename for file in encryption_extension):
```
these two cases should be mutually exclusive.